### ISSUE TITLE
Release new versions

### ISSUE THREAD

**Issue by paulmillr** [micromatch/is-glob#11]
@jonschlinkert hey, would be great to release is-glob 4.0.1 and is-extglob 2.1.2; tiny RAM usage improvements with the var out-of-scope.

**Comment by jonschlinkert** [comment]
@paulmillr agreed. We might also be able to simplify this speed it up by using negative lookbehinds in the regex. I can't remember when support for lookbehinds was added to node. I'll look into it when I get more time. 

Either @doowb or I (or someone else with NPM rights) will give you rights to publish if I can't get to this before this weekend. My main priority is getting micromatch published, but I will be out of the office 
 a lot this week.

**Comment by phated** [comment]
I can get this cut as a small release once I get npm publish rights.

**Comment by phated** [comment]
Waiting on CI

**Comment by phated** [comment]
Travis is finally done. Published as 4.0.1
